### ISSUE TITLE
ciphertext cannot be decrypted using that key

### ISSUE THREAD

**Issue by jpdagostin0** [etesync/ios#105]
Hello, I followed the guide for a production setup of an etebase server. I created a user in the django admin interface and then setup the account with a password etc. on the webclient. I tried logging in using the same credentials on iOS, and I get the error, "ciphertext cannot be decrypted using that key". The authentication works, and I can read/write data on the web interface. I think this may be an issue with the iOS app on the app store. Thanks!

**Comment by rmetschke** [comment]
I see the exact same error message in iOS after the OS has been erased and EteSync freshly installed. However, I am not self hosting; I pay EteSync/Base to host. I haven’t seen any updates in this repo for years. I can’t help but wonder if this is going to be fixed at all. I may need to find somewhere else to host contacts and calendars as this is untenable.

**Comment by jpdagostin0** [comment]
I’m looking to move to NextCloud, the platform has e2ee and support for calendars, contacts, notes, etc.

**Comment by ubuntophe** [comment]
I have the same error message trying to setup EteSync for contacts and calendar on iOS 17.5.1.
@jpdagostino, are you sure NextCloud supports e2ee for calendars and contacts?

**Comment by trymeouteh** [comment]
I been having this issue as well

**Comment by ubuntophe** [comment]
I found an alternative. For 1€ per month, the privacy-focused german service posteo.de offers e2ee calendars and contacts compatible with Caldav and Cardav. I use it on iOS and thunderbird (Linux).

**Comment by rmetschke** [comment]
Do you know if it supports shared calendars and shared contacts?

**Comment by ubuntophe** [comment]
No, I dont know. To be checked on their website.

**Comment by Morpheus0x** [comment]
I get the same error. There haven't been any commits to this repo since 2022 and the last update of the iOS App was in 2020. I use EteSync on Android and Linux and hoped to also get it to work on my iPad. Hopefully this will be fixed sometime in the future, but I don't expect it will. I totally understand that maintaining the clients for that many platforms is very time-consuming, and the iOS App is probably not at all a priority.

**Comment by nguyenkvvn** [comment]
I get this error too. Looks like I'm not the only one. I will make another account and share my main account into it as a workaround I guess. :(

**Comment by nguyenkvvn** [comment]
No dice with the alt account. This seems to be an issue with the app itself... :/

**Comment by ivendotdev** [comment]
Is the iOS app abandoned? I just subscribed to Etesync thinking I can use it on my phone.

Non-self hosted.

**Comment by nguyenkvvn** [comment]
@ivendotdev  I have given up on EteSync unfortunately. I concur that the platform itself just seems abandoned.

**Comment by ivendotdev** [comment]
> [@ivendotdev](https://github.com/ivendotdev) I have given up on EteSync unfortunately. I concur that the platform itself just seems abandoned.

Indeed, that's too bad. I suppose I'll cancel my subscription since not being able to use it on my phone is a deal breaker. Thanks for the heads up.
